packages/core/src/__tests__/ceramic-api.test.ts

```diff
@@ -392,10 +402,6 @@ describe('Ceramic API', () => {
           streamId: streamA.id,
           paths: ['/b/d', '/notExistStreamId'],
         },
-        {
-          streamId: notExistStreamId,
```

Comment:
why was this removed?

Reply:
Good catch, thanks. With go-ipfs somehow timeout is not respected and the test hangs. Worrying. Created an issue to track the fix.